<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>625</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>.AppleSystemUIFont</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="east">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>460</y>
      <width>101</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>east</string>
    </property>
   </widget>
   <widget class="QPushButton" name="west">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>west</string>
    </property>
   </widget>
   <widget class="QPushButton" name="north">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>420</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>north</string>
    </property>
   </widget>
   <widget class="QPushButton" name="south">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>500</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>south</string>
    </property>
   </widget>
   <widget class="QPushButton" name="map">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>450</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>map</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text1">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>20</y>
      <width>520</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>420</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>character</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="jdt">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
    <property name="format">
     <string>房间探索进度%p%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tp">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>480</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>tp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="harb">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>510</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Introduction</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>211</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="npc">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>420</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>npc</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HuangBo">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>480</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>HuangBo</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
